spec/unit/uimanager_bench.lua

```diff
@@ -109,9 +109,8 @@ describe("UIManager schedule massive collision tests", function()
     end
 end)
 
-
-describe("UIManager schedule massive rediculous tests", function()
-    print("Doing massive rediculous collision tests ......... this takes really a lot time")
+describe("UIManager schedule massive ridiculous tests", function()
+    print("Doing massive ridiculous collision tests ......... this takes really a lot of time")
```

Comment:
If you're changing those lines for a minor typo you might as well correct the grammatical crimes and add in some stylistic improvements. ;-)

```suggestion
describe("UIManager schedule massive ridiculous tests", function()
    print("Performing massive ridiculous collision tests ......... this really takes a lot of time")
```

Reply:
So shall it be. 
(As a non English speaking, I appreciate any hint to improve my language skills :-))
 tbc.